tachyon/math/base/batch_inverse_benchmark.cc

```diff
@@ -24,7 +24,7 @@ void BM_BatchInverse(benchmark::State& state) {
   std::vector<F> fields = base::CreateVector(
       state.range(0), [](size_t i) { return F::FromBigInt(BigInt(i + 1)); });
   for (auto _ : state) {
-    CHECK(F::BatchInverseInPlace(fields));
+    static_cast<void>(F::BatchInverseInPlace(fields));
```

Comment:
This is an old pattern. include `<tuple>` and change like below. See also See https://stackoverflow.com/a/77195811

```suggestion
    std::ignore = F::BatchInverseInPlace(fields);
```